addons/calendar/wizard/calendar_provider_sync_wizard.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="calendar_provider_sync_wizard_view_form" model="ir.ui.view">
+        <field name="name">calender.calendar.provider.sync.wizard.view.form</field>
+        <field name="model">calendar.calendar.provider.sync.wizard</field>
+        <field name="arch" type="xml">
+            <form js_class="provider_sync_wizard">
+                <field name="external_calendar_provider" widget="radio" options="{'horizontal': true}"/>
+                <div class="o_setting_right_pane" attrs="{'invisible': [('external_calendar_provider', '!=', 'google')]}">
+                    <img alt="Google Calendar icon" src="/calendar/static/src/img/google_calendar_40.png" style="height: 40px; margin-right: 5px"/>
```

Comment:
inline style? :(

Reply:
This allowed to avoid the empty space caused by empty table rows. I guess it's barely visible (except on narrow viewports), and not worth this complexity.